<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SITN 4D - Liste des modifications</string>
  </property>
  <widget class="QTableWidget" name="tableSMT">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>851</width>
     <height>192</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::DoubleClicked</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="columnCount">
    <number>7</number>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>851</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Propriété de la selection</string>
   </property>
   <widget class="QCheckBox" name="groupHistorizeCheckbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enregistrer dans la base temporelle</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QComboBox" name="groupTypeCombobox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="groupDescriptionLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>731</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="applyGroupSettingsToSelectedButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Appliquer aux lignes sélectionnées</string>
    </property>
   </widget>
   <widget class="QPushButton" name="applyGroupSettingsToAllButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>130</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Appliquer à toutes les lignes</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>361</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:#1E90FF;</string>
   </property>
   <property name="text">
    <string>✔️ Valider toutes les modifications</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
